neurovault/apps/statmaps/tasks.py

```diff
@@ -174,6 +174,65 @@ def generate_glassbrain_image(image_pk):
         img.thumbnail.save("glass_brain_%s.jpg" % img.pk, content_file)
         img.save()
 
+# SURFACE IMAGE GENERATION ###########################################################################
+
+@shared_task
+def generate_surfacce_image(image_pk):
+    from neurovault.apps.statmaps.models import Image
+    from scipy.io import loadmat
+    import numpy.matlib as matlib
+    from scipy.interpolate import interpn
+
+    img = Image.objects.get(pk=image_pk)
+    img_vol = nib.load(img.file.path)
+    data_vol = img_vol.get_data()
+    if data_vol.ndim > 3:
+        data_dim = data_vol.shape[3]  #number of time points
+    else:
+        data_dim = 1
+    this_path = os.path.abspath(os.path.dirname(__file__))
+
+    for hemi in ['lh', 'rh']:
+        if hemi == 'lh':
+            ras_coor = loadmat(os.path.abspath(os.path.join(this_path, "static", "anatomical",
+                                                            "lh.avgMapping_allSub_RF_ANTs_MNI2fs.mat")))['ras']
```

Comment:
Consider replacing the if statement by a string interpolation:
`"%s.avgMapping_allSub_RF_ANTs_MNI2fs.mat" % hemi`